cypress/e2e/tests/pages/explorer/workloads/pods.spec.ts

```diff
@@ -52,9 +68,18 @@ describe('Pods', { testIsolation: 'off', tags: ['@explorer', '@adminUser'] }, ()
     });
 
     it('pagination is visible and user is able to navigate through pods data', () => {
+      cy.tableRowsPerPage(10);
+      cy.reload(true);
+      // The pods count returned just before the assertions are out of sync with the created pods during the setup
+      // Waiting seems to mitigate this sync issue. This behavior seems consistent/reproducible on HA Rancher
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(30000);
```

Comment:
move this test to the bottom to run last.. that will give sufficient time to load the 25 resources so you can remove the wait here

Reply:
Still happens too quick, the other tests are not too time consuming.

Reply:
i tried rearranging locally and its working for me. i think its worth a try to rearrange the tests and if it turns out to be flaky we can revisit. we should avoid using waits were we can as much as possible

Reply:
i get the feeling that either
- cypress isn't correctly waiting for the api requests that create the pods to finish
- there is lag between the api requests to create the pods and them entering the cache to be returned in the next request for all /v1/pods
  - @moio it would be good to rule this out, or understand the lag. if pods are created quickly via api we would expect them to appear in the cache in milliseconds correct, and not seconds? Are there automated tests for this already?

api / cache lag aside there are other options
1. fetch the initial pods count by making a request to v1/pods and capturing `count`. create pods. cy.wait(() => check captured count + created pods count === current v1/pods count
2. the same as above but using v1/counts --> data.counts.pods.summary.count

Reply:
@richard-cox yes the expectation is milliseconds

@brudnak do your tests cover this case?

Reply:
@yonasberhe23  @richard-cox I agree we should avoid using waits _**when possible**_ in this case the tests were **_very Flaky_** and this was the only option I found to mitigate the issue. 

Running the suite isolated from other tests isn't a reliable way to spot flakyness overall.

Reply:
@moio my case for pagination & limit uses secrets

Reply:
https://github.com/rancher/dashboard/issues/11648

Reply:
@brudnak resource type should not matter - they are all processed in the same (or very similar) ways.

Do your tests specifically exercise reading-immediately-after-write, or did you ever observe any substantial delays between writing and reading back?